### ISSUE TITLE
daemon: global ringbuf loss is misattributed or omitted from session evidence

### ISSUE THREAD

**Issue by gnanirahulnutakki** [ArdurAI/ardur#219]
## Summary

The process-lifecycle consumer keeps one daemon-global `CaptureLoss` accumulator. A malformed record increments `loss.RingbufDropped`; the next valid event is passed to `processKernelEvent`, and the accumulator is then reset unconditionally.

Current flow:

```go
loss.RingbufDropped++
...
d.processKernelEvent(evt, loss)
loss = kernelcapture.CaptureLoss{}
```

`processKernelEvent` writes a receipt only when `routeEvent` correlates that valid event to a session. This creates two correctness failures:

1. If the next valid event is uncorrelated, the global loss count is reset without appearing in any session receipt.
2. If it is correlated, the entire host-global loss count is attached to whichever session happened to produce the next valid event, even though the malformed records cannot be attributed to that session.

Found while completing the operator documentation requested by #90.

## Impact

Evidence completeness can be understated for active sessions or arbitrarily charged to one session. Daemon warnings remain accurate, but session receipts are not a durable or correctly scoped representation of the host-level capture gap.

## Design constraint

Do not merely retain the count until the next correlated event; that avoids omission but still misattributes daemon-global loss to one session. Model the loss as a global capture-gap epoch/window, then make every affected active session status/receipt account for that gap, or expose a daemon-global evidence status that verifiers are required to compose with session claims.

## Acceptance criteria

- malformed-record loss is persisted or retained until durably represented
- an uncorrelated valid event cannot erase the loss signal
- concurrent active sessions receive honest, non-arbitrary evidence degradation
- tests cover malformed -> uncorrelated -> correlated flow and multiple active sessions
- operator docs describe the final attribution contract

## Evidence

- `go/cmd/ardur-kernelcaptured/daemon_linux.go`: global accumulator and unconditional reset
- `go/cmd/ardur-kernelcaptured/main.go`: `processKernelEvent` returns early for uncorrelated events
- `go/pkg/kernelcapture/correlator.go`: receipt verdict degrades only when loss reaches `EventContext`
